Test/git-issues/git-issue-505.dfy

```diff
@@ -0,0 +1,45 @@
+// RUN: %dafny /compile:0 /timeLimit:20 "%s" > "%t"
```

Comment:
If these tests are timing out as it is, can we change the /timeLimit to 1 second? That may also make the tests more deterministic.

I also note that all 5 assertions below are marked with "timed out", but the .expect file only shows 3 of them timing out. Perhaps reducing the time limit to 1 second would make all of them time out.

Reply:
Then, at least adjust the comments below.

Reflecting on this again, the purpose of these tests is to show that Dafny produces a reasonable error message when there is a time out. For that purpose, making use of bitvector conversions works today, because Z3 does these slowly for wide words. Such a test could then just be one of the lemmas below and the /timeLimit flag can be set as small as possible. The purpose of these tests is not to check that Z3 is able to do some tests quickly enough, so I propose those be removed.

Reply:
Per discussion - comments fixed, but tests remain